### DISCUSSION TITLE
Datetime Series

### DISCUSSION THREAD

**Discussion by pabloseren** [GAA-UAM/scikit-fda#322] [Q&A] [answered]
Hi! i've been playing around with the library and i cant find the way to represent my datetime series, i've tried with several type of datetime object but none work.

So basically the question is, can i work with this type of date to create a FDataGrid?
i also tried with numpy array, and the problem seems to be with the datetime object (or pandas timespan etc)

![image](https://user-images.githubusercontent.com/65686960/107850740-f38d9980-6e04-11eb-97d5-cd39af7ba045.png)

TypeError: unsupported operand type(s) for +: 'datetime.datetime' and 'float'

thanks again!

**Comment by vnmabus** [comment, marked as answer]
I have to say that I have never explicitly added support for datetimes, just floating point arrays. That said, numpy recently added support for dates using its `datetime64` dtype (https://numpy.org/doc/stable/reference/arrays.datetime.html), so MAYBE, that could work, or at least we could try to add support for this type.

**Reply by pabloseren** [reply]
i just tried and apparently is not working with np.datetime64 either 
if you ever had to, how did you handle datetime series?, i am considering time expresed as integers, since they are equally separeted.
Thanks you in advance

**Reply by vnmabus** [reply]
I would express time in seconds/hours/days/whatever as a floating point number.

**Reply by pabloseren** [reply]
thanks! will see how that works.
